codebuild/spec/buildspec_fuzz_scheduled.yml

```diff
@@ -0,0 +1,87 @@
+---
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+# this file except in compliance with the License. A copy of the License is
+# located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing permissions and
+# limitations under the License.
+version: 0.2
+
+batch:
+  build-matrix:
+    static:
+      env:
+        privileged-mode: true
+    dynamic:
+      env:
+        compute-type:
+          - BUILD_GENERAL1_LARGE
+        image:
+          - 024603541914.dkr.ecr.us-west-2.amazonaws.com/docker:ubuntu22codebuild
+        privileged-mode: true
+        variables:
+          S2N_LIBCRYPTO: 
+            - awslc
+          FUZZ_TESTS:
+            - "s2n_cert_req_recv_test"
+            - "s2n_certificate_extensions_parse_test"
```

Comment:
Are they listed manually in the existing codebuild job? I think they are, which means this is at least meeting the existing bar. You can open a separate issue for doing it better :)

Reply:
Created a new issue for this in #4769